great_expectations_cloud/agent/agent.py

```diff
@@ -456,11 +456,15 @@ def _update_status(self, job_id: str, status: JobStatus, org_id: UUID) -> None:
         )
         with create_session(access_token=self.get_auth_key()) as session:
             data = status.json()
-            session.patch(agent_sessions_url, data=data)
+            response = session.patch(agent_sessions_url, data=data)
             LOGGER.info(
                 "Status updated",
                 extra={"job_id": job_id, "status": str(status), "organization_id": str(org_id)},
             )
+        if not response.ok:
+            raise RuntimeError(  # noqa: TRY003 # one off error
+                "Status Update action had an error while connecting to GX Cloud."
+            )
```

Comment:
Can you double-check that we return a 200 here and not a 204. Any 2XX response should be OK since we don't rely on the payload

Reply:
This is a `204` https://github.com/greatexpectationslabs/mercury/blob/b78b06b47c3a3707bab35096d70922ffafd4f6f9/services/ge_cloud/mercury/handlers/rest/agent_v0.py#L213-L214